Leaderboard/bin/www

```diff
@@ -3,88 +3,94 @@
 /**
  * Module dependencies.
  */
-
+require('babel/register');
 var app = require('../app');
+var db = require('../database').default;
 var debug = require('debug')('Leaderboard:server');
 var http = require('http');
 
-/**
- * Get port from environment and store in Express.
- */
+db().then(function() {
```

Comment:
The database needs to be set up before certain parts of the application are set up. Since initializing the database doesn't block (seriously node?), we have to do the database setup prior to setting up the entire application, otherwise we run the risk of a potentially vulnerable race condition.